casbin/config/config.py

```diff
@@ -98,6 +98,31 @@ def _write(self, section, line_num, b):
 
         del b[:]
 
+# lookups up the value using the provided key and converts the value to a bool
+    def getBool(self, key):
+        return self.get(key).capitalize() == "True"
+
+# lookups up the value using the provided key and converts the value to a int
+    def getInt(self, key):
+        return int(self.get(key))
+
+# lookups up the value using the provided key and converts the value to a float
+    def getFloat(self, key):
+        return float(self.get(key))
+
+# lookups up the value using the provided key and converts the value to a string
+    def getString(self, key):
+        return self.get(key)
+
+# lookups up the value using the provided key and converts the value to an array of string
```

Comment:
@divyagar Please write comments in the function body. Refer to [Specification](https://www.python.org/dev/peps/pep-0257/#id14).

Then, it's better to add unit tests .